test/testrunner.vcxproj

```diff
@@ -29,6 +29,7 @@
     <ClCompile Include="..\cli\filelister.cpp" />
     <ClCompile Include="..\cli\threadexecutor.cpp" />
     <ClCompile Include="..\lib\astutils.cpp" />
+    <ClCompile Include="..\lib\library.cpp" />

```

Comment:
That does not work because the cppcheck-core library is already linked in. It results in duplicate symbols